### PR TITLE
Show and Tell: Improve tracking hours badge contrast

### PR DESCRIPTION
**Pull request by pjeweb**
## Describe your changes

In presentation view, make the Give an Hour tracking display have more contrast.

## Notes for testing your change

Look at it.

### CODE REVIEW

**Comment by MattIPv4** [comment]
Thoughts on this?

image

image

```tsx
          <>
            {` — `}
            <Link
              href="/show-and-tell/give-an-hour"
              className={classes(
                "text-green-700 inline-flex items-center gap-1",
                isPresentationView
                  ? "group text-nowrap rounded-full bg-blue-900/95 p-1 shadow-lg transition-all hover:scale-102 hover:bg-blue-900 focus:bg-blue-900 text-3xl hover:text-green-600"
                  : "hover:underline focus:underline",
              )}
              target="_blank"
              custom
            >
              <strong
                className={classes(
                  "bg-gradient-to-br to-green-600 bg-clip-text font-bold text-transparent",
                  isPresentationView
                    ? "from-blue-500 transition-colors group-hover:from-blue-400 group-hover:to-green-500 pl-2 leading-none"
                    : "from-blue-800",
                )}
              >
                Gave {hours} {hours === 1 ? "Hour" : "Hours"} for Earth
              </strong>
              <IconWorld className={classes("inline-block", isPresentationView ? "h-12 w-12" : "h-8 w-8")} />
            </Link>
          </>
```

**Comment by pjeweb** [comment]
> Thoughts on this?

Very clean.

File: apps/website/tailwind.config.ts [hunk 1]
```diff
@@ -63,6 +63,7 @@ const config = {
         700: "#3B5A8F",
         800: "#2A4168",
         900: "#1A2840",
+        950: "#0c1626",
```
Comment (MattIPv4) [inline apps/website/tailwind.config.ts:66]: This should no longer be needed with the new commit?

Comment (pjeweb) [reply apps/website/tailwind.config.ts]: yep

**Review by MattIPv4 — APPROVED** [review]

**Pull request merged**